### DISCUSSION TITLE
曹冲

### DISCUSSION THREAD

**Discussion by MiG-2233** [share121/inter-knot#1559] [其他]
在这里放一个可爱的冲儿，大家不要古锭刀酒杀他噢
![ad4e1fd638c385a158f79ed44caaf4c4db4f3325_raw](https://github.com/user-attachments/assets/02420278-c457-4080-ac36-46c5b228f4f4)

**Comment by Sui-Qing** [comment]
wc，蒸qwp

**Comment by shenguojia123** [comment]
聪以知远，明以察微

**Comment by shenguojia123** [comment]
见微知著，识人心智

**Comment by dominic121-dot** [comment]
你说得对，但是吾虽年迈，箭矢犹锋。

**Reply by Jekothalep** [reply]
冷

**Comment by cooperatior** [comment]
这我记得曹称冲象

**Comment by deguodiguoyuanshuai** [comment]
能冲🐘吗
